examples/sycl/00_pvc_gemm/00_pvc_gemm.cpp

```diff
@@ -225,17 +248,27 @@ struct ExampleRunner {
     auto problem_shape_MNKL = cute::append<4>(problem_size, 1);
     auto [M, N, K, L] = problem_shape_MNKL;
 
+    // Padded values
+    int N_B = cute::ceil_div(N, AlignElemB) * AlignElemB;
```

Comment:
there is a `cute::round_up` which I think does what you are doing here

Reply:
replaced